docs/variants/dell_optiplex/initial-deployment.md

```diff
@@ -18,91 +18,69 @@ for following models
 
     ![](../../images/case_open2.jpg)
 
-3. Now, it is time to release the disk dock. Lift up the handle of the CD/DVD drive
+1. Now, it is time to release the disk dock. Lift the handle of the CD/DVD drive
    bay.
 
     ![](../../images/disk_dock_open1.jpg)
 
-4. Pull the CD/DVD drive bay to the CPU fan side.
+1. Pull the CD/DVD drive bay to the CPU fan side.
 
     ![](../../images/disk_dock_open2.jpg)
 
-5. Move the blue disk dock handle to the CPU fan side.
+1. Move the blue disk dock handle to the CPU fan side.
 
     ![](../../images/disk_dock_open3.jpg)
 
-6. The screw should be at the more giant hole now. Lift up the whole dock to
+1. The screw should be at the giant hole now. Lift the whole dock to
    remove it.
 
     ![](../../images/disk_dock_open4.jpg)
 
-7. When the dock is removed, the service mode jumper should be visible.
+1. When the dock is removed, the service mode jumper should be visible.
 
     ![](../../images/overview_service_jumper.jpg)
 
-8. Place the jumper in the place marked by the red rectangle.
+1. Place the jumper in the place marked by the red rectangle.
 
     ![](../../images/service_jumper_header.jpg)
 
-9. It should look like this.
+1. It should look like this.
 
     ![](../../images/service_jumper.jpeg)
 
-10. Power on the machine. You should see a warning that the service jumper is
+1. Power on the machine. You should see a warning that the service jumper is
 active. Press F1 to proceed and boot to your Linux system.
 
     ![](../../images/service_mode_warn.jpg)
 
 ## OS booting
 
-For simplicity we using network boot [netboot.xyz from USB](https://netboot.xyz/docs/booting/usb).
+For simplicity, we are using
+[Dasharo Tools Suite](../../../common-coreboot-docs/dasharo_tools_suite#bootable-usb-stick)
+booted over network.
 
-Boot system from USB:
-
-* From section `Tools` choose `Utilities (64-bit)`
-* Then from section `netboot.xyz tools` choose `Kernel cmdline params`
-* You should see prompt `Enter kernel cmdline parameters`
-* Type: `iomem=relaxed` and ++enter++
-* Use ++esc++ to get back to netboot.xyz main menu
-* From section `Distributions` choose `Live CDs`, then `Debian`, `Debian Live
-  11 (bullseye)` and `Debian 11 (Core)`
-
-**NOTE**: If trustworthiness of that solution is in question please note
-netboot.xyz can be [self-hosted](https://netboot.xyz/docs/selfhosting).
-
-## Flashrom installation
-
-* Install flashrom v1.1 or newer with your distribution's package manager if
-  you don't have it installed yet. If your distro doesn't provide flashrom or
-  provides an outdated one, you can build it yourself using
-  [this instruction](https://www.flashrom.org/Downloads).
-* Or compile recent version of flashrom:
-
-  ``` console
-  sudo apt install libpci-dev libftdi-dev libusb-1.0-0-dev
-  git clone https://github.com/flashrom/flashrom.git
-  cd flashrom
-  sudo make install
-  ```
+* Attach a wired network cable to the device's Ethernet port.
+* Boot from USB and enter the shell.
```

Comment:
So over network or USB? Above we say about network. How do we boot over network? I beleive USB is used for initial deployment.

Reply:
@macpijan ideally I would like to relevant DTS documentation. I assume we need network connection to run our canonical use case for deployment which means HCL report and BIOS backup.

Reply:
so we still ned to update this section

Reply:
@macpijan https://github.com/Dasharo/docs/pull/139/commits/f312a418169a77003a833dab9ca2cf3c2db4e606

Reply:
LGTM, as in previous comment. Closing